samples/django/django_user/myoauth/urls.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2016 Google Inc.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from django.conf.urls import include, url
+from django.contrib import admin
+import django.contrib.auth.views
```

Comment:
[Use `from <package hierarchy> import <module>`](https://google.github.io/styleguide/pyguide.html#Imports).

... unless there's such a thing as a Django style guide that is in conflict with the Google Python Style Guide _and_ it's more important in these examples to demonstrate Django-compliant style?